site/docs/design_tokens/border-radius.mdx

```diff
@@ -0,0 +1,51 @@
+---
+name: Border Radius
+menu: Design tokens
+route: /design-tokens/border-radius
+---
+
+import LargeParagraph from "../../src/components/LargeParagraph";
+
+
+# Border Radius
+
+<LargeParagraph>The border-radius tokens are used to give elements rounded corners.</LargeParagraph>
+
+The border-radius CSS property rounds the corners of an element's outer edge. The radius applies to the background element, even if the element has no border.
+
+## Principles
+
+When creating components or editing existing ones, keep in mind, that rounded corners, whether small or big, **should not be applied to any part of the Helsinki brand assets**. 
+
+## Usage
+
+### When to use
+
+- **Visual distinction from the surrounding elements.** As humans, we are conditioned to favor rounded corners, while sharp corners invoke a feeling of danger.
+- **When designing for a very young demographic.** As children, we quickly learn that sharp corners hurt and that rounded corners are safer.
+- **Reproducing natural or some cultural connotations.** Rounded corners can be found everywhere in nature and humans have – throughout history – tried to replicate shapes found in nature.
+
```

Comment:
E.g. 
Use rounded corners to make a distinction between buttons and tags/keywords.
Use rounded corners in elements that are commonly known with rounded borders, e.g. media controls.
Use rounded corners when emphasizing human communication, e.g. in chat speech bubbles.

Reply:
Jes, lisätty